Formula/marathon-swift.rb

```diff
@@ -0,0 +1,27 @@
+class MarathonSwift < Formula
+  desc "Makes it easy to write, run and manage your Swift scripts 🏃"
+  homepage "https://github.com/JohnSundell/Marathon"
+  url "https://github.com/JohnSundell/Marathon/archive/1.0.1.tar.gz"
+  sha256 "647b098fc30af9d597d359dfb3e1aa2f62e29aa06aa4ffc6d417d1b9f9e54081"
+
+  depends_on :xcode => ["8.3", :build]
+
+  def install
+    ENV["SDKROOT"] = MacOS.sdk_path
+
+    if MacOS::Xcode.version >= "9.0"
+      system "swift", "package", "--disable-sandbox", "update"
+      system "swift", "build", "-c", "release", "-Xswiftc", "-static-stdlib", "--disable-sandbox"
+    else
+      ENV.delete("CC") # https://bugs.swift.org/browse/SR-3151
+      system "swift", "package", "update"
+      system "swift", "build", "-c", "release", "-Xswiftc", "-static-stdlib"
+    end
+
+    bin.install ".build/release/marathon"
+  end
+
+  test do
+    system "#{bin}/marathon", "help"
```

Comment:
This needs to be a test that exercises and verifies the functionality of the software once installed.